umap/static/umap/js/modules/importer.js

```diff
@@ -1,104 +1,112 @@
+    this.fileBox = DomUtil.create('div', 'formbox', this.container)
+    this.fileInput = DomUtil.element({
       tagName: 'input',
       type: 'file',
       parent: this.fileBox,
       multiple: 'multiple',
       autofocus: true,
     })
-    this.urlInput = L.DomUtil.element({
+    this.urlInput = DomUtil.element({
       tagName: 'input',
       type: 'text',
       parent: this.container,
-      placeholder: L._('Provide an URL here'),
+      placeholder: translate('Provide an URL here'),
     })
-    this.rawInput = L.DomUtil.element({
+    this.rawInput = DomUtil.element({
       tagName: 'textarea',
       parent: this.container,
-      placeholder: L._('Paste your data here'),
+      placeholder: translate('Paste your data here'),
+    })
+    DomUtil.element({
+      tagName: 'h4',
+      parent: this.container,
+      textContent: translate('Import from:'),
     })
-    this.typeLabel = L.DomUtil.add(
+    this.typeLabel = DomUtil.add(
       'label',
       '',
       this.container,
-      L._('Choose the format of the data to import')
+      translate('Choose the format of the data to import')
     )
-    this.layerLabel = L.DomUtil.add(
+    this.layerLabel = DomUtil.add(
       'label',
       '',
       this.container,
-      L._('Choose the layer to import in')
+      translate('Choose the layer to import in')
     )
-    this.clearLabel = L.DomUtil.element({
+    this.clearLabel = DomUtil.element({
       tagName: 'label',
       parent: this.container,
-      textContent: L._('Replace layer content'),
+      textContent: translate('Replace layer content'),
       for: 'datalayer-clear-check',
     })
-    this.submitInput = L.DomUtil.element({
+    this.submitInput = DomUtil.element({
       tagName: 'input',
       type: 'button',
       parent: this.container,
-      value: L._('Import'),
+      value: translate('Import'),
       className: 'button',
     })
     this.map.help.button(this.typeLabel, 'importFormats')
-    this.typeInput = L.DomUtil.element({
+    this.typeInput = DomUtil.element({
       tagName: 'select',
       name: 'format',
       parent: this.typeLabel,
     })
-    this.layerInput = L.DomUtil.element({
+    this.layerInput = DomUtil.element({
       tagName: 'select',
       name: 'datalayer',
       parent: this.layerLabel,
     })
-    this.clearFlag = L.DomUtil.element({
+    this.clearFlag = DomUtil.element({
       tagName: 'input',
       type: 'checkbox',
       name: 'clear',
       id: 'datalayer-clear-check',
       parent: this.clearLabel,
     })
-    L.DomUtil.element({
+    DomUtil.element({
       tagName: 'option',
       value: '',
-      textContent: L._('Choose the data format'),
+      textContent: translate('Choose the data format'),
       parent: this.typeInput,
     })
-    for (let i = 0; i < this.TYPES.length; i++) {
-      option = L.DomUtil.create('option', '', this.typeInput)
+    for (let i = 0, option; i < this.TYPES.length; i++) {
```

Comment:
```suggestion
    for (const type of this.TYPES) {
```

And the use `type` below?

Reply:
:+1: